gradle/project.gradle.kts

```diff
@@ -92,14 +92,10 @@ allprojects {
             os.isWindows -> "win"
             else -> failIO { "Unable to determine javafx dependency classifier due to unfamiliar system=$os" }
          }
-         implementation("org.openjfx:javafx-base:$version:$classifier")
-         implementation("org.openjfx:javafx-controls:$version:$classifier")
-         implementation("org.openjfx:javafx-graphics:$version:$classifier")
-         implementation("org.openjfx:javafx-fxml:$version:$classifier")
-         implementation("org.openjfx:javafx-media:$version:$classifier")
-         implementation("org.openjfx:javafx-swing:$version:$classifier")
-         implementation("org.openjfx:javafx-web:$version:$classifier")
-
+         val jfxLibs = arrayOf("base", "controls", "graphics", "fxml", "media", "swing", "web")
+         for (lib in jfxLibs) {
```

Comment:
This is actually less declarative than before, just pointing out. Can stay this way.

Reply:
maybe less declarative, but much neater, because you get an overview of all javafx packages in use in just one line, whereas previously there was a lot of repeated information which was just clutter imo.

Reply:
Well the point is to see the dependencies, not javafx modules, right?

Reply:
Well the modules are the dependencies in this case, but I guess that's a matter of personal preference.